src/helper/core/goe-api.ts

```diff
@@ -34,3 +36,24 @@ export async function createImpl(env: NodeJS.ProcessEnv): Promise<Api> {
     }
   };
 }
+
+function getGasIncPct(): number {
+  const envKeys = Object.keys(process.env);
+
+  const patterns = [
```

Comment:
What is the purpose of using multiple envs for one value?

Reply:
For fuzzy matching, this parameter is considered to be added for roughly these keywords.